doc/manual.md

```diff
@@ -6549,16 +6549,16 @@ The `include` statement can be used outside the top level, as such:
 Module names in imports
 -----------------------
 
-A module alias can be introduced via the `as` keyword:
+A module alias can be introduced via the `as` keyword, and the original module name
+is not accessible then:
```

Comment:
I would still prefer the pre-change text, but if you're adamant about putting both statements together, I feel that this is the best that can be done:
```suggestion
A module alias can be introduced via the `as` keyword, replacing the original module's name in the current global scope:
```

Reply:
Why the change?

1. this sentence (or this piece of information) is strongly coupled with the first paragraph of the section, but not the current one (2nd paragraph).
2. most important information is now delivered fast and efficiently with a single glance.

I do not prefer your suggestion. Why?

1. Use the original wording as much as possible.
2. A thing can't be _replaced_ if it does exist at all.
3. "current global scope" adds to the burden of understanding: why _current_? why _global_? how about _local_?

Reply:
```suggestion
A module alias can be introduced via the `as` keyword, after which the original module name
is inaccessible:
```